plugins/inputs/traefik/README.md

```diff
@@ -0,0 +1,46 @@
+# Telegraf Input Plugin: Traefik
+
+This plugin gather health check status from services registered in Traefik.
+
+### Configuration:
+
+```toml
+# Description
+[[inputs.traefik]]
+  ## Required Traefik server address (default: "127.0.0.1")
+  server = "127.0.0.1"
+  ## Required Traefik port (default "8080")
+  port = 8080
+  ## Required Traefik instance name (default: "default")
+  instance = "default"
```

Comment:
A user can already set a custom tag to differentiate their server like this:
```toml
[[inputs.traefik]]
  tags = {"instance": "default"}
```

I suggest automatically adding a tag called "url" with the actual url, this is consistent with other plugins are doing.  With this automatic tag, it is probably worth changing the `server` option to a list so that it is easy to scrape multiple servers.  This is how the nginx, apache, ping, haproxy, etc plugins are working.